docs/walkthroughs/setup-mobile-on-existing-cluster.adoc

```diff
@@ -0,0 +1,85 @@
+== Setup Mobile on your existing OpenShift cluster
+
+This document will outline the prerequisites and steps required to try out
+the OpenShift mobile services and mobile client integrations on an existing cluster.
+
+
+=== Pre Requisites 
+
+- Your cluster should be running OpenShift 3.9 or greater
+- The service catalog should be installed
+- The Automation Broker should be installed
```

Comment:
worth making this a link to the broker docs here? https://github.com/openshift/ansible-service-broker#getting-started-on-openshift

Reply:
We could maybe link to the repos, but I don't really want to get questions on how to set these bits up. Aiming the doc at people who have already set this stuff up